gtd-timer/Startup.cs

```diff
@@ -94,7 +94,7 @@ public void ConfigureServices(IServiceCollection services)
             }
             );
 
-            services.AddSwaggerGen(c =>
+            /*services.AddSwaggerGen(c =>
```

Comment:
why are we doing this?

Reply:
to disable swagger on production? there are smarter ways to do this, the simplest being using config. investigate this